opm/parser/eclipse/Deck/Section.hpp

```diff
@@ -47,7 +47,7 @@ class Section : public DeckView {
 
         // returns whether the deck has all mandatory sections and if all sections are in
         // the right order
-        static bool checkSectionTopology(const Deck& deck,
```

Comment:
Oh, you mean making the return type non-const, sure. Yeah, that (obviously) works.